src/main/features/offer/routes/response.ts

```diff
@@ -51,9 +48,8 @@ export default express.Router()
             await OfferClient.rejectOffer(user)
             res.redirect(Paths.rejectedPage.evaluateUri({ externalId: user.claim.externalId }))
             break
-
           default:
-            res.redirect(Paths.responsePage.evaluateUri({ externalId: user.claim.externalId }))
+            throw new Error(`Option ${form.model.option} is not supported`)
```

Comment:
I prefer hard error instead of 'refresh' redirect which makes it easier to spot problems. When counter offer is implemented it will go away.